test/torchaudio_unittest/functional/functional_impl.py

```diff
@@ -45,6 +45,24 @@ def test_shape(self, shape):
         output_waveform = F.lfilter(waveform, a_coeffs, b_coeffs)
         assert shape == waveform.size() == output_waveform.size()
 
+    def test_9th_order_filter_stability(self):
+        """
+        the filter coefficients is get by `scipy.signal.butter(9, 850, 'hp', fs=22050, output='ba')`
+        """
+        x = torch.zeros(1024, dtype=self.dtype, device=self.device)
+        x[0] = 1
+        a = torch.tensor([1., -7.60545606, 25.80187885, -51.23717435,
+                          65.62093428, -56.20096964, 32.18274279, -11.88025302,
+                          2.56506938, -0.24677075], dtype=self.dtype, device=self.device)
+        b = torch.tensor([0.49676025, -4.47084227, 17.88336908, -41.72786118,
+                          62.59179178, -62.59179178, 41.72786118, -17.88336908,
+                          4.47084227, -0.49676025], dtype=self.dtype, device=self.device)
+        y = F.lfilter(x, a, b, False)
+        if self.dtype == torch.float64:
+            assert torch.all(y.abs() < 0.8)
```

Comment:
You should be able to use ```self.assertLess(y.abs, 0.8)``` here, which will give a better error if this test where to fail.